tests/test_object.py

```diff
@@ -516,3 +516,56 @@ def __init__(
     with pytest.raises(RuntimeError):
         with pytest.warns(RuntimeWarning):
             BadObject(0, 1).to_yaml_s()  # cannot borrow missing attribute
+
+
+class MyObjectWithKwargs(audobject.Object):
+
+    @audobject.init_decorator(
+        borrow={
+            'borrow_arg': 'dict',
+        },
+        hide=['hide_arg', 'unassign_arg'],
+        resolvers={
+            'resolve_arg': audobject.resolver.Tuple,
+        }
+    )
+    def __init__(
+            self,
+            arg: str,
+            **kwargs,
```

Comment:
Maybe change to:

```suggestion
            *,
            kwarg: str,
            **kwargs,
```
?